File: dominio/modelos orijinales/GoMenuOption.java
package servicebackmenu2;

public class GoMenuOption extends MessagesOption{
    private ListMenus listMenus;    
    
    public GoMenuOption(String title) {
        super(title);
        this.listMenus = ListMenus.getInstance();
    }
    
    public void interact() {
        this.listMenus.getMenu(this.title).interact();
    }
}

File: conect4/src/main/resources/static/js/memus/models/generics/GoMenuOption.js

import Option from "./Option.js";
import ListMenus from "./ListMenus.js";
import Message from "../../../Messages/Message.js";

class GoMenuOption extends Option {

    constructor(title) {
        super();
       this.menuname = title;
        this.listMenus = ListMenus.getInstance();
    }
    
    updateTitle() {
        this.title = Message.getInstance().getMessages(this.menuname).title;
    }
    
    execute() {
        return this.listMenus.getMenu(this.menuname);
    }
    
}

export default GoMenuOption;